src/js/components/SearchBar.jsx

```diff
@@ -73,12 +74,12 @@ export class SearchBar extends PureComponent {
       <div>
         {/* search field */}
         <TextField
```

Comment:
I imagine you've considered using the material UI autocomplete field to do this?
http://www.material-ui.com/#/components/auto-complete